internal/server/requests_test.go

```diff
@@ -0,0 +1,57 @@
+package server
+
+import (
+        "io/ioutil"
+        "net/http"
+        "testing"
+        "time"
+        "github.com/stretchr/testify/assert"
+	"strings"
+	"net/url"
+)
+
+func TestLanuchHandleRequests(t *testing.T){
+	go HandleRequests(":8080")
```

Comment:
This is not the right way to write tests. Take a look at [this materials](https://github.com/sgrzywna/golang-academy/tree/main/workshop_1#testing.)